### PR TITLE
[AMORO-2731]: auto import antd-vue component & icons

### PR DESCRIPTION
**Pull request by zhangmo8**
## Why are the changes needed?

auto import antd-vue component & icons
Fix the dark theme style

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by zhangmo8** [comment]
#2731

**Review by tcodehuber — COMMENTED** [review]
LGTM, but I have left some comments, PTAL.

File: amoro-ams/amoro-ams-dashboard/src/views/404.vue [hunk 1]
```diff
@@ -30,13 +30,9 @@ limitations under the License.
 
 <script lang="ts">
 import { defineComponent } from 'vue'
-import { Button as AButton } from 'ant-design-vue'
 
 export default defineComponent({
   name: 'Page404',
```
[the remark this answers is not in the document]
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/views/404.vue:35]: What ?

File: amoro-ams/amoro-ams-dashboard/src/components/tables-sub-menu/TablesMenu.vue [hunk 2]
```diff
@@ -115,12 +108,8 @@ interface ITableItem {
 export default defineComponent({
   name: 'TablesMenu',
   components: {
-    // PlusOutlined,
-    SearchOutlined,
-    CloseCircleOutlined,
     CreateDBModal,
     virtualRecycleScroller,
```
[the remark this answers is not in the document]
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/components/tables-sub-menu/TablesMenu.vue:112]: Maybe we should improve the eslint rules and fix rules after the migration is completed.

[the remark this answers is not in the document]
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/components/tables-sub-menu/TablesMenu.vue:112]: #2852

File: amoro-ams/amoro-ams-dashboard/src/views/404.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhoujinsong) [reply amoro-ams/amoro-ams-dashboard/src/views/404.vue:35]: @tcodehuber seems to suggest removing the not-needed comma(,) here, but we can do it in another PR.
I noticed you have created #2852 to track it.

**Comment by zhoujinsong** [comment]
Thanks a lot for the work! @zhangmo8 

I tested it in my local environment and found some issues to be fixed:

1. The terminal page does not show the shortcut correctly:
image

2. The login page components seem bigger than the old version:
New version:
image

Old version:
image

File: amoro-ams/amoro-ams-dashboard/src/views/404.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/views/404.vue:35]: Yes, this is indeed an interesting proposal. In some front-end projects, it is necessary to keep the last comma as a specification.

[the remark this answers is not in the document]
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/views/404.vue:35]: I didn't add or remove the comma in this submission, which I understand is due to the developers' inconsistent eslint configuration. So after completing the migration, I will unify the eslint specification in another issues and improve the eslint fix

**Comment by zhangmo8** [comment]
image

**Review by zhoujinsong — APPROVED** [review]
LGTM.

Thanks for the  contribution!

**Review by tcodehuber — APPROVED** [review]
LGTM

**Pull request merged**